src/components/seo.tsx

```diff
@@ -5,9 +5,10 @@ import React from 'react';
 interface Props {
   title: string;
   description: string;
+  canonical?: string;

```

Comment:
Didn't you say you wanted all pages to have a canonical meta tag? Then we shouldn't have this prop be optional

Reply:
Test

Reply:
test

Reply:
@alexmonteiroqccareer

Reply:
@davewelsh  ok. I will fix it.